src/components/advanced-search/advanced-search-dates.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <v-dialog id="advanced-search-dates" v-model="dialog" persistent max-width="44rem">
+    <v-card height="32rem" class="py-6">
+      <v-row>
+        <v-col>
+          <span class="title-bold-16" :class="{ 'error-message': validate && !hasStartDate }">Select Start Date:</span>
+          <v-date-picker
+            id="start-date-calendar"
+            class="mt-2"
+            v-model="startDateText"
+            :max="currentDateString"
+            elevation="5"
+          >
```

Comment:
I'm not up to speed on the Date situation, i just used the code written for the Edit UI with regards to retrieving the current server date. 

Set to store in the same manner in `App.vue` and we just retrieve it.
![Screen Shot 2021-05-12 at 8 02 02 AM](https://user-images.githubusercontent.com/53542131/117998010-573a3c80-b2f8-11eb-8e6d-d165c71999bc.png)

Is this an issue in all the apps, or is something done differently here?

Reply:
Gaah, I totally missed line 11... Here, I _think_ this code is as good as it gets, though it may still confuse a user in a different timezone wondering wht their start/end date doesn't find a NR one day before/after... In other places we've been explicit that our system dates/times are "Pacific time". Is it possible to indicate that here? Maybe check with Scott/Tracey.

Yes, this timezone thing is an issue in all the apps, though I've been picking away at it bit by bit. (And I'm not thrilled about my current solution but we can talk about that separately.)

Reply:
I think we are explicit to "Pacific time" in this instance as well. When we set the date in these pickers we use this method.

![Screen Shot 2021-05-12 at 9 23 58 AM](https://user-images.githubusercontent.com/53542131/118010483-f1ec4880-b303-11eb-97f2-842ad06546d6.png)

Reply:
Yes, our system works in Pacific time, but we don't tell the user that on the date picker.

Reply:
ahh i see.